lxqt-admin-user/mainwindow.cpp

```diff
@@ -40,6 +41,7 @@ MainWindow::MainWindow():
     connect(ui.actionChangePasswd, SIGNAL(triggered(bool)), SLOT(onChangePasswd()));
     connect(ui.actionRefresh, SIGNAL(triggered(bool)), SLOT(reload()));
 
+    connect(ui.tabWidget,SIGNAL(currentChanged(int)),SLOT(onTabChange(int)));
```

Comment:
Please, add the `#if defined(Q_OS_FREEBSD)` here.  There is no need to execute noop functions in runtime of other platforms.